### PR TITLE
updating during the import, backward comaptibility

### PR DESCRIPTION
**Pull request by vlinhart**
when the import is started second time, it's an update of data

test against python 2.7 and django1.11 which is still supported

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 SECRET_KEY = "!2*nm%ps%x8!ykyb^s9+!l1vcmeh+(f&de%br=js*7(5i_rmet"
 
 # needed since travis uses Ubuntu 14.04
-if distro.linux_distribution() == ('Ubuntu', '16.04', 'Xenial Xerus'):
+if distro.linux_distribution() == ('Ubuntu', '16.04', 'Xenial Xerus') or \
+    distro.linux_distribution() == (u'Linux Mint', u'18.3', u'Sylvia'):
```
Comment (eracle) [inline tests/settings.py:16]: Interesting

**Review by eracle — COMMENTED** [review]
Thank you very much for the PR.
I haven't tested it, nor I will, since I have no possibility doing it right now, you can always test it by running tox.
By the way, everything seems ok, it respects the style, and adds back-compatibility.
Thank you very much

File: tests/settings.py (hunk 1 above)
Comment (vlinhart) [reply tests/settings.py:16]: yeah it wouldn't use the sqlite geo extension for me without this...

Comment (eracle) [reply tests/settings.py:16]: So, is not a general rule, is a specific setting tailored on your system's characteristics, what about having a simple flag instead?

Comment (vlinhart) [reply tests/settings.py:16]: Hard to tell, seem at least these older Ubuntus need the variable `SPATIALITE_LIBRARY_PATH` set. Hard to tell if it breaks something for another distros.

https://code.djangoproject.com/ticket/26137

Comment (eracle) [reply tests/settings.py:16]: ok, thanks

**Pull request merged**